package.json

```diff
@@ -2,7 +2,7 @@
   "name": "storytime-frontend",
   "version": "0.0.0",
   "scripts": {
-    "dev": "cross-env APP_ENV=development vite",
+    "dev": "cross-env APP_ENV=development vite --host",
```

Comment:
I'm not sure --host is needed here. I don't think it is hurting anything being there, but you are accessing the test env via "http://localhost:8081" and as I understand it, --host binds the instance to the IP address of the computer it's running on and makes it available on the local network.